doc/man7/openssl-glossary.pod

```diff
@@ -66,6 +169,35 @@ This is specified in RFC 5208:
 
 L<https://tools.ietf.org/html/rfc5208>
 
+=item Property
+
+A property is a way of classifying and selecting algorithm implementations.
```

Comment:
"classifying" comes off a bit odd.  An *operation*, like `EVP_SIGNATURE`, could be viewed as a classification, I don't see properties in that way.  I would remove `classifying and`

Reply:
I actually disagree. Properties to me are very much about classifying, i.e. the set of all algorithm implementations that are FIPS validated, or the set of all algorithms that are in the default provider, the set of algorithms that implement blinding. These are all different ways of classifying algorithms and the way to do that is through properties. Dropping "classifying and" misses out half the purpose of what properties are used for. One is to classify algorithm implementations, and one is to select algorithm implementations based on those classifications.